apps/prairielearn/src/lib/features/manager.sql

```diff
@@ -59,7 +59,7 @@ ON CONFLICT (
 SET
   enabled = EXCLUDED.enabled;
 
--- BLOCK disable_feature
+-- BLOCK delete_feature
```

Comment:
👀 This was a bug, `manager.ts` refers to delete_feature, not disable_feature.

Reply:
Any thoughts of including the opposite feature (find blocks that are referenced but not found) to find these bugs?